Jenkinsfile

```diff
@@ -5,7 +5,7 @@ pipeline {
     triggers { cron(cron_string) }
     environment {
         SERVICE = 'ocpmetal/assisted-service'
-        ISO_CREATION = 'installer-image-build'
+        ISO_CREATION = 'assisted-iso-create'
```

Comment:
maybe you should change the build step as well? I mean the makefile and `build-assisted-iso-generator-image`